lib/errors.ts

```diff
@@ -21,7 +21,10 @@ import { TypedError } from 'typed-error';
 import { getChalk, stripIndent } from './utils/lazy';
 import { getHelp } from './utils/messages';
 
-export class ExpectedError extends TypedError {}
+export class ExpectedError extends TypedError {
+	public code?: string;
+	public exitCode?: number;
+}
```

Comment:
I guess this is for error code support?  Shouldn't we add the new properties to `TypedError`?

Reply:
Ok, I think we should do that at some point, but we can switch to that later I guess.